src/app/service/register.service.ts

```diff
@@ -64,4 +64,19 @@ export class RegisterService {
       });
 
   }
+  checkbox(user_id: string) {
+    var headers = new Headers();
+    headers.append('Content-Type', 'application/json');
```

Comment:
this is not the correct place to do this. it should be global

Reply:
ok sir